tests/Unit/Plugins/CacheTest.php

```diff
@@ -105,3 +105,27 @@
 
     expect($cache->get($key))->toBeNull();
 });
+
+it('can flush the cache', function (): void {
+    $dir = __DIR__.'/../../.peck-test-flush.cache';
```

Comment:
This means the new directory is in the `tests/` directory correct?
That's at least what I would expect